looker_blueprints/download_look_as_file.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+import sys
+import os
+import shipyard_utils as shipyard
+try:
+    import helpers
+except BaseException:
+    from . import helpers
+try:
+    import exit_codes as ec
+except BaseException:
+    from . import exit_codes as ec
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--base-url', dest='base_url', required=True)
+    parser.add_argument('--client-id', dest='client_id', required=True)
+    parser.add_argument('--client-secret', dest='client_secret', required=True)
+    parser.add_argument('--look-id', dest='look_id', required=True)
+    parser.add_argument('--destination-file-name', dest='dest_file_name', required=True)
+    parser.add_argument('--destination-folder-name', dest='dest_folder_name', required=False)
+    parser.add_argument('--file-type', dest='file_type',
+                        choices=['json','txt','csv', 'json_detail', 'md', 'xlsx'
+                                 'sql', 'png', 'jpg'],
+                        type=str.lower,
+                        required=True)
+    args = parser.parse_args()
+    return args
+
+
+def download_look(look_sdk, look_id, file_format):
+    try:
+        # Options are csv, json, json_detail, txt, html, md, xlsx, sql (raw query), png, jpg
+        response = look_sdk.run_look(
+        look_id=look_id,
```

Comment:
Need to handle look_id not existing with a specific error.

Reply:
This is done